cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -244,3 +247,21 @@ Then("There is a shadow effect between the parent row and first subrow", () => {
       "rgba(0, 20, 29, 0.1) 0px 6px 4px -4px inset"
     );
 });
+
+When("I have a large viewport", () => {
+  cy.viewport(700, 345);
+});
+
+When("I have a small viewport", () => {
+  cy.viewport(700, 240);
+});
+
+When("pageSize is selected", () => {
+  flatTablePageSizeSelect().click();
+});
+
+Then("pageSizeSelectList is visible at the {word}", (position) => {
+  flatTablePageSelectListPosition()
```

Comment:
```suggestion
  flatTablePageSelectListPosition().should("have.attr", "data-popper-placement", position)
```

Reply:
As discussed, I left the format as it is to keep it uniform with the other steps in the file. I logged a ticket to standardise all step definition files as their formats and indents differ.